### ISSUE TITLE
Feature Request: Infer from .stache

### ISSUE THREAD

**Issue by leoj3n** [stealjs/steal-bundler#44]
According to the README, and in my experience, this bundler will only infer from `.less` and `.js`.

For whatever reason, I expected `bundleAssets` in steal-tools to also infer from `.stache`, but this currently does not seem to be the case.

I'm surprised to not see any open issues here about inferring from `.stache`; I thought this would be a popular assumption (and therefore feature request). Am I mistaken in thinking that?

Is this even a valid feature request? Thanks.

EDIT:

For myself and anyone else in the future having a similar need to include images specified in `.stache` HTML image tags like `<img src="img/pic.png" />`, modify your `build.js` to be like:

```diff
diff --git a/client/build.js b/client/build.js
index dd5e573..b68b69d 100644
--- a/client/build.js
+++ b/client/build.js
@@ -4,6 +4,9 @@ const buildPromise = stealTools.build(
   {},
   {
     minify: false,
-    bundleAssets: true,
-  }
+    bundleAssets: {
+      infer: true,
+      glob: ['img/**/*'],
+    },
+  },
 );
```

Now the entire contents of the `./img` folder are included in `./dist/img`! You will have to update this to glob any images in other locations as well.

However, it would still be nice to use `steal-bundler` to infer only what is specified in `.stache`. 👍

**Comment by matthewp** [comment]
Yeah, people have requested it before. It depends on whether we need to rewrite the urls in the stache files. If not then it shouldn't be too hard just to do the inference. If we need to rewrite urls then it's likely not doable because by the time steal-bundler runs the stache has already been converted to javascript and bundled.

**Comment by leoj3n** [comment]
@matthewp Only thing I can think to do in that case is make it clear in the README (probably also in steal-tools because nobody is looking here) and include an example of using `bundleAssets` glob.
